hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/compact/TestInlineCompaction.java

```diff
@@ -294,8 +294,9 @@ public void testCompactionRetryOnFailureBasedOnTime() throws Exception {
       moveCompactionFromRequestedToInflight(instantTime, cfg);
     }
 
-    // When: commit happens after 10s
-    HoodieWriteConfig inlineCfg = getConfigForInlineCompaction(5, 10, CompactionTriggerStrategy.TIME_ELAPSED);
+    // When: commit happens after 1000s. assumption is that, there won't be any new compaction getting scheduled within 100s, but the previous failed one will be
```

Comment:
is this gonna add a lot to the running time?

Reply:
nope. we don't want any new compaction to be scheduled in that 2nd phase only. thats why. So, no hit on the test run time.